app/styles/main.scss

```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```

Comment:
Yeah, I agree. This solution is not maintainable. I'll keep the sticky footer in the back off my mind though, maybe there's a better way to do it.